conversation/python/sdk/conversation/requirements.txt

```diff
@@ -1 +1,3 @@
-dapr>=1.15.0
\ No newline at end of file
+#dapr>=1.15.0
+
+-e ../../../../../python-sdk
```

Comment:
We need to set the version per quickstart and this should pull in the latest python sdk rc: [v1.16.0rc1](https://github.com/dapr/python-sdk/releases/tag/v1.16.0rc1)

Reply:
sure, this is all temporary to work in my computer, we need python sdk with [my changes](https://github.com/dapr/python-sdk/pull/822) that is not merged yet